forest/bonsai/simulate_gps_data.py

```diff
@@ -23,6 +33,65 @@
 restaurant = [42.3679,-71.1089]
 R = 6.371*10**6
 
+
+def get_path(coords1: Tuple[float], coords2: Tuple[float], transport: str,
+             api_key: str) -> Tuple[np.ndarray, float]:
+    """Calculates paths between sets of coordinates
+
+    This function takes 2 sets of coordinates and
+    a mean of transport and using the openroute api
+    calculates the set of nodes to traverse
+    from location1 to location2 along with the duration
+    and distance of the flight.
+    Args:
+        coords1: coordinates of start point (lat, lon)
+        coords2: coordinates of end point (lat, lon)
+        transport: means of transportation,
+            can be one of the following:
+            (car, bus, foot, bicycle)
+        api_key: api key collected from
+            https://openrouteservice.org/dev/#/home
+    Returns:
+        path_coordinates: 2d numpy array containing [lon,lat] of route
+        distance: distance of trip in meters
+    """
+
+    lat1 = coords1[0]
+    lon1 = coords1[1]
+    lat2 = coords2[0]
+    lon2 = coords2[1]
+
+    if great_circle_dist(lat1, lon1, lat2, lon2) < 250:
```

Comment:
For better readability and to avoid redundant function calls it probably makes sense to save the value: `distance = great_circle_distance(...)`